core/routemgmt/createRoute.js

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
This file looks like an exact copy of the cloudant create document action.

https://github.ibm.com/BlueMix-Fabric/bluewhisk/blob/master/blue/catalog/cloudant/database-actions/create-document.js

It would be nice to not clone code.   Did you consider either using the cloudant package directly, or delegating interactions with cloudant to the cloudant package via whisk.invoke or a sequence?